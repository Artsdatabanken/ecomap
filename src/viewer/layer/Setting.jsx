import React from 'react'
import {Divider} from 'material-ui'

const Setting = ({children}) =>
  <div>
    <div style={{display: 'inline-block', marginTop: '8px', height: '58px'}}>
      {children}
    </div>
    <Divider />
  </div>

export default Setting
