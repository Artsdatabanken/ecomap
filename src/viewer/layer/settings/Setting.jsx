import React from 'react'
import Label from './Label'

const Setting = ({title, children}) =>
  <div>
    <div style={{display: 'inline-block'}}>
      <Label>
        {title}
      </Label>
      {children}
    </div>
  </div>

export default Setting
