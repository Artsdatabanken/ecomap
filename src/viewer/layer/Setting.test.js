import React from 'react'
import renderer from 'react-test-renderer'
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import muiTheme from '../../theme_jest'

import Setting from './Setting'

it('renders ok', () => {
  const tree = renderer.create(
    <MuiThemeProvider muiTheme={muiTheme}>
      <Setting><div /></Setting>
    </MuiThemeProvider>
)
  expect(tree.toJSON()).toMatchSnapshot()
})
