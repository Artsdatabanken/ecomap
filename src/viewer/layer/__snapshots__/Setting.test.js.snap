// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ok 1`] = `
<div>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "58px",
        "marginTop": "8px",
      }
    }
  >
    <div />
  </div>
  <hr
    style={
      Object {
        "backgroundColor": "#e0e0e0",
        "border": "none",
        "height": 1,
        "margin": 0,
        "marginLeft": 0,
        "marginTop": -1,
        "muiPrepared": true,
      }
    }
  />
</div>
`;
